src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -178,6 +182,13 @@ public void start(Promise<Void> startPromise) {
                 this.router.errorHandler(HttpResponseStatus.NOT_FOUND.code(), this::errorHandler);
 
                 this.router.route("/metrics").handler(this::metricsHandler);
+                MeterRegistry meterRegistry = BackendRegistries.getDefaultNow();
```

Comment:
@ppatierno Aren't we doing this step already in `Application.java` class? Shouldn' we just directly use 
```
    if (metricsReporter.getMeterRegistry() != null) {
          ......
     }
```

Reply:
good catch @ShubhamRwt ! I have just pushed the change.